lib/types/api.types.ts

```diff
@@ -295,6 +304,90 @@ export type KnownSDK =
   | AppExtensionSDK
   | HomeExtensionSDK
 
+export interface BaseAppSDK {
+  /** @deprecated since version 4.0.0 consider using the CMA instead
```

Comment:
I wouldn't put spaceAPI anymore for anyone using AppSDK, wdyt?

Reply:
wouldn't that make it a breaking change? Or is it fine because it is still possible to use the BaseExtensionSDK? 🤔

Reply:
I agree that would make sense, but with the aliasing now I don't think that is so straight forward anymore 😬